tf2onnx/onnx_opset/math.py

```diff
@@ -544,3 +544,26 @@ def version_11(cls, ctx, node, **kwargs):
             cast_back_node.set_attr("to", dtypes[0])
             ctx.set_dtype(cast_back_node.output[0], dtypes[0])
             ctx.copy_shape(node.name, cast_back_node.output[0])
+
+@tf_op("MatrixInverse")
+class Inverse:
+
+    @classmethod
+    def version_12(cls, ctx, node, **kwargs):
+        utils.make_sure(node.get_attr('adjoint').i == 0, "adjoint must be false")
+        shapes = node.output_shapes
+        dtypes = node.output_dtypes
+        ctx.remove_node(node.name)
+        ctx.make_node("Inverse", inputs=node.input, outputs=node.output, name=node.name,
+                      shapes=shapes, dtypes=dtypes)
```

Comment:
A good trick I found is to use the onnx_op= parameter -- I think the snippet below should work the same way, and no need to remove + add nodes.

```python
@tf_op("MatrixInverse", onnx_op="Inverse")
class Inverse:
    @classmethod
    def version_12(cls, ctx, node, **kwargs):
        utils.make_sure(node.get_attr('adjoint').i == 0, "adjoint must be false")
```

Reply:
Good point, thanks!